### PR TITLE
[Metricbeat] Separate vpc metricset to vpn, transitgateway and natgateway metricsets

### PR DESCRIPTION
**Pull request by kaiyan-sheng**
## What does this PR do?

This PR is to separate vpc metricset into three smaller metricsets: vpn, transitgateway and natgateway metricsets. This PR will also add dashboards for each metricset.

## Why is it important?

This will help with users who only wants to enable one or two out of all three metricsets.

## Checklist

- [x] My code follows the style guidelines of this project
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] I have made corresponding change to the default configuration files
- [x] I have added tests that prove my fix is effective or that my feature works

## Screenshots
vpn dashboard:
Screen Shot 2020-03-18 at 12 21 41 PM (2)

nat gateway dashboard:
Screen Shot 2020-03-17 at 3 29 25 PM (2)

transit gateway dashboard:
Screen Shot 2020-03-18 at 1 39 46 PM (2)

### CODE REVIEW

File: x-pack/metricbeat/module/aws/_meta/kibana/7/dashboard/Metricbeat-aws-trasitgateway-overview.json [hunk 1]
```diff
@@ -0,0 +1,853 @@
...
+              "y": 29
+            },
+            "panelIndex": "b141f90b-739e-46f3-83c9-9c4661183837",
+            "panelRefName": "panel_5",
+            "title": "Bytes Dropped - no route",
+            "version": "7.6.1"
+          },
+          {
+            "embeddableConfig": {
+              "title": "Packet Dropped - no route"
```
Comment (mtojek) [inline x-pack/metricbeat/module/aws/_meta/kibana/7/dashboard/Metricbeat-aws-trasitgateway-overview.json]: s/Packet/Packets/

File: x-pack/metricbeat/module/aws/_meta/kibana/7/dashboard/Metricbeat-aws-trasitgateway-overview.json [hunk 2]
```diff
@@ -0,0 +1,853 @@
...
+            "gridData": {
+              "h": 12,
+              "i": "c6a76f92-248b-4cae-a03f-7d34d58098ae",
+              "w": 24,
+              "x": 24,
+              "y": 29
+            },
+            "panelIndex": "c6a76f92-248b-4cae-a03f-7d34d58098ae",
+            "panelRefName": "panel_6",
+            "title": "Packet Dropped - no route",
```
Comment (mtojek) [inline x-pack/metricbeat/module/aws/_meta/kibana/7/dashboard/Metricbeat-aws-trasitgateway-overview.json]: s/Packet/Packets/

File: x-pack/metricbeat/module/aws/_meta/kibana/7/dashboard/Metricbeat-aws-trasitgateway-overview.json [hunk 3]
```diff
@@ -0,0 +1,853 @@
...
+              "y": 41
+            },
+            "panelIndex": "1d08d3b8-3bd7-4f90-854d-be08cb119273",
+            "panelRefName": "panel_7",
+            "title": "Bytes Dropped - black hole",
+            "version": "7.6.1"
+          },
+          {
+            "embeddableConfig": {
+              "title": "Packet Dropped - black hole"
```
Comment (mtojek) [inline x-pack/metricbeat/module/aws/_meta/kibana/7/dashboard/Metricbeat-aws-trasitgateway-overview.json]: s/Packet/Packets/

File: x-pack/metricbeat/module/aws/_meta/kibana/7/dashboard/Metricbeat-aws-trasitgateway-overview.json [hunk 4]
```diff
@@ -0,0 +1,853 @@
...
+              "y": 41
+            },
+            "panelIndex": "40e82e50-b30c-40eb-bbee-9bbfc3d3311f",
+            "panelRefName": "panel_8",
+            "title": "Packet Dropped - black hole",
+            "version": "7.6.1"
+          }
+        ],
+        "timeRestore": false,
+        "title": "[Metricbeat AWS] TransitGateway Overview",
```
Comment (mtojek) [inline x-pack/metricbeat/module/aws/_meta/kibana/7/dashboard/Metricbeat-aws-trasitgateway-overview.json]: s/Packet/Packets/

File: x-pack/metricbeat/module/aws/_meta/kibana/7/dashboard/Metricbeat-aws-trasitgateway-overview.json (hunk 1 above)
Comment (kaiyan-sheng) [reply x-pack/metricbeat/module/aws/_meta/kibana/7/dashboard/Metricbeat-aws-trasitgateway-overview.json]: @mtojek Good point, thanks for catching them. I just fixed them in the latest commit.

**Pull request merged**